src/api-reference/financial-integration/v4.financial-documents-schema.markdown

```diff
@@ -139,6 +139,249 @@ Name|Type|Format|Description
+`entryOrgUnit1Value` through `entryOrgUnit6Value`| `CustomField`| - |The details from the custom fields. These fields may not have data, depending on the configuration. If configured as a list, this field contains the name of the list item. Maximum length 64 characters.
+`entryOrgUnit1Code` through `entryOrgUnit6Code`|`CustomField`| - |The details from the custom fields. These fields may not have data, depending on the configuration. Maximum length 48 characters.
+`expenseTypeCode`|`String`| - |Alphanumeric. Code for the expense type. Maximum length 5 characters.
+`payrollData`|`Array`| Array of [PayrollData](#scpayrolldata) |Represents payroll related data for an expense entry
+
+### <a name="scpayrolldata"></a>Payroll - ExpenseEntry - Payroll Data
+
+Name|Type|Format|Description
+---|---|---|---
+`taxableType`|`String`|-|Determines if taxable or non-taxable.
+`taxationType`|`String`|-|Determines what type of taxation (eg: flat rate, individual, Non taxable) for taxable entries
+`entryApprovedAmount`|`Number`|-|Maximum 23 digits to the left of the decimal; maximum 8 digits to the right of the decimal; with a maximum of 23 digits for the entire field.|Currency. Amount approved in the reimbursement currency.
+`wageType`|`String`|-|The wage type required by the payroll system for posting
+`payrollPayIndicator`|`String`|-| Indicates whether the amounts due employee will be reimbursed through payroll or not. possible values Y or N. If N, then amounts transferred to payroll are for taxation record-keeping purposes only.  If Y, then amounts transferred to payroll are for reimbursement AND taxation record-keeping purposes
+
+### <a name="scprlcaa"></a>Payroll - CashAdvanceUtilizationData
+
+Name|Type|Format|Description
+---|---|---|---
+`cashAdvanceApplicationAmount`|`Number`|Maximum 23 digits to the left of the decimal; maximum 8 digits to the right of the decimal; with a maximum of 23 digits for the entire field.|Currency. Cash advance utilized amount.
+`cashAdvanceClearingWageType`|`String`|***|***
+`payrollPayIndicator`|`String`|Y/N|This field indicates whether the expense group the report user belongs to is configured to reimburse employees using Payroll.
+`cashAdvanceReturnsAmount`|`Number`|Maximum 23 digits to the left of the decimal; maximum 8 digits to the right of the decimal; with a maximum of 23 digits for the entire field.|Currency. Amount of cash advance returned.
+
+### <a name="scprltnryrsltdtl"></a>Payroll - ItineraryResultDetail
+
+Name|Type|Format|Description
+---|---|---|---
+`itineraryHeader`|`Object`|[Payroll - ItineraryHeader](#scprlItnryHdr)|Header with general information about the itinerary
+`dailyAllowanceDetails`|`Array`| Array of [Payroll - DailyAllowanceDetail](#scprldlyallwnceDtl)|List of summarized and detailed daily travel allowance calculation results of the itinerary. 
+`itineraryResultHeader`|`Object`| [Payroll - ItineraryResultHeader]($scprlitnryRsltHdr)|General data of the itinerary
```

Comment:
This link creates a 404 error. See below:
"404 - page not found
The 
6322377c98f77a5a7f86fd2ee548617c63bdac8c

 branch of 
preview.developer.concur.com

 does not contain the path 
src/api-reference/financial-integration/$scprlitnryRsltHdr."

Reply:
This no longer creates a 404 error. However, it is now a dead link. Please fix.

Reply:
Fixed. Tested and works.